### ISSUE TITLE
Queue Special Issues Questions

### ISSUE THREAD

**Issue by lomky** [department-of-veterans-affairs/caseflow#14524]
## Description
The research in #14433 led to a couple more open questions we need to answer

## Acceptance criteria
 - [ ] Clarify expectations of Special Issues Selection in Queue
 - [ ] Confirm behavior of Special Issues List & Legacy Special Issues noted in #14433 

## Background/context/resources
[Caseflow-Dispatch](https://github.com/department-of-veterans-affairs/caseflow/wiki/Caseflow-Dispatch)  
[slack convo](https://dsva.slack.com/archives/CQCUMFA0H/p1591977760151300)
[Special Issues in Queue Issue](https://github.com/department-of-veterans-affairs/caseflow/issues/6854), [PR](https://github.com/department-of-veterans-affairs/caseflow/pull/6871)

**Comment by ajspotts** [comment]
[Resulting investigation](https://hackmd.io/BiwHywxcTeePht2N6NMFOQ?view)
[Epic for Dispatch deprecation](https://github.com/department-of-veterans-affairs/caseflow/issues/10024)

TLDR: 
- Special Issues exist in both the Dispatch and Queue applications however their presence in each serve different purposes.  
  - SIs in Dispatch are added by dispatch users and play a role in determining the routing of legacy appeals depending on which SIs are selected
  - SIs in Queue are added by attorneys & judges and at the time of this writing exist only for reporting purposes
- Part of the potential future deprecation process of the Dispatch app would need to be the implementation of the routing functionality into the Queue app
